notebooks/Text_Extensions_for_Pandas_Overview.ipynb

```diff
@@ -0,0 +1,1819 @@
+{
```

Comment:
<p>This section seems a bit off theme to the overall NLP focus of this notebook. Maybe a line explaining a potential use of TensorArray might be helpful. Or any other rationale behind adding TensorArray support to TEP. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>